checkmates/pipelines/transformers.py

```diff
@@ -83,6 +84,55 @@ def _get_feature_provenance(self):
         return {}
 
 
+"""Component that normalizes skewed distributions using the Yeo-Johnson method"""
+
+
+class SimpleNormalizer(Transformer):
+    """Normalizes skewed data according to the Yeo-Johnson method.
+
+    Args:
+        impute_strategy (string): Impute strategy to use. Valid values include "mean", "median", "most_frequent", "constant" for
```

Comment:
This docstring seems to be wrong, not an imputer here.

Reply:
fixed and removed args as they are not applicable here